### ISSUE TITLE
dat-ecosystem website_terminal

### ISSUE THREAD

**Issue by Mehrabbruno** [dat-ecosystem/organization#105]
#70

## `@todo`

- [x] design terminal
   - `@input` 📦 [terminal_frame_v0.0.5](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1156%3A2960&t=2eJNNRFFInrn2WRG-4) from #86 
   - [x] design a terminal container
   - [x] add tabs for each terminal 
   - [x] add icon btn with `x` icon
   - [x] add icon btn for maximum and minimum
   - [x] update task and record worklog
   - `@output` 📦 [`terminal_v0.0.1`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1156%3A3400&t=2eJNNRFFInrn2WRG-4) from [comment](https://github.com/dat-ecosystem/organization/issues/105#issuecomment-1451041838)

---

- [x] update terminal_v0.0.1
   - `@input` 📦  [`terminal_v0.0.1`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1156%3A3400&t=2eJNNRFFInrn2WRG-4)
   - [x] move the tabs to bottom of the terminal 
   - [x] change the color of the terminal
   - [x] convert icon btns to be commands
   - [x] update task and record worklog
   - `@output` 📦 [`terminal_v0.0.2`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1211-1911&t=yzkyPlOhuzbjUi62-4) from [comment](https://github.com/dat-ecosystem/organization/issues/105#issuecomment-1461101694)

---

- [x] update terminal_v0.0.2
   - `@input` 📦 [`terminal_v0.0.2`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1211-1911&t=yzkyPlOhuzbjUi62-4)
   - [x] bring the v0.0.1 UX back to the updated design
   - [x] change the color of the terminal
   - [x] add a scrollbar
   - [x] update task and record worklog
   - `@output` 📦 [`terminal_v0.0.3`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1273-3879&t=If4neK4QfpiquEnl-4) from [comment](https://github.com/dat-ecosystem/organization/issues/105#issuecomment-1478325671)

---

- [x] update terminal_v0.0.3
   - `@input` 📦 [`terminal_v0.0.3`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1273-3879&t=If4neK4QfpiquEnl-4)
   - [x] update the buttons and the icons
   - [x] add border and change the color of the terminal tabs
   - [x] update the scrollbar
   - [x] update task and record worklog
   - `@output` 📦 [`terminal_v0.0.4`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1315-1911&t=oHWrNNsurk7EuHVg-4) from [comment](https://github.com/dat-ecosystem/organization/issues/105#issuecomment-1487840069)

---

- [x] update terminal_v0.0.4
   - `@input` 📦 [`terminal_v0.0.4`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1315-1911&t=oHWrNNsurk7EuHVg-4)
   - `@input` 📦 [`scrollbar_v0.0.1`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1360-6666&t=Ox51Ohm1m5cMB0nq-4) from #108 
   - [x] update the UI to be flat retro
   - `@output` 📦 [`terminal_v0.0.5`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1359-2762&t=Ox51Ohm1m5cMB0nq-4) from [comment](https://github.com/dat-ecosystem/organization/issues/105#issuecomment-1500712204)

---

- [x] update terminal_v0.0.5
   - `@input` 📦 [`terminal_v0.0.5`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1359-2762&t=Ox51Ohm1m5cMB0nq-4)
   - `@input` 📦 [`scrollbar_v0.0.2`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1360-7963&t=Ox51Ohm1m5cMB0nq-4) from #108 
   - [x] add the scrollbar_v0.0.2.
   - [x] change the highlight colors
   - [x] make the buttons more compacts
   - [x] record and update tasks, worklog etc
   - `@output` 📦 [`terminal_v0.0.6`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1359-3226&t=Ox51Ohm1m5cMB0nq-4) from [comment](https://github.com/dat-ecosystem/organization/issues/105#issuecomment-1500712204)

---

- [x] update terminal_v0.0.6
   - `@input` 📦 [`terminal_v0.0.6`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1359-3226&t=Ox51Ohm1m5cMB0nq-4)
   - [x] read feedback
   - [x] remove the extra buttons from the bottom tab contianer
   - [x] record and update tasks, worklog etc
   - `@output` 📦 [`terminal_v0.0.7`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1430-2686&t=idhOGERipmYR9EQg-4) from [comment](https://github.com/dat-ecosystem/organization/issues/105#issuecomment-1508324311)

**Comment by Mehrabbruno** [comment]
### Tasks

- design terminal - `1h17min`
   - design a terminal container
   - add tabs for each terminal
   - add icon btn with x icon
   - add icon btn for maximum and minimum
   - update task and record worklog - `9min`
   - @output [terminal_v0.0.1](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1156%3A3400&t=2eJNNRFFInrn2WRG-4)

---

### Worklog

[worklog-117](https://youtu.be/Iek_wA_ZWZM)

---


### Feedback

---

### Proposals

**Comment by serapath** [comment]
**feedback `2023.03.02`**

notice the tab bar has fine lines. looks crisp. retro systems did not have such high resolutions, so things looked a big "bolder" maybe?
maybe tabs have same color as the terminal itself?


![win1](https://i.iinfo.cz/images/64/gui04-11.png)



love everything, just thinking if we can find inspiration and subtle ways to make it look more retro :-) 


![win2](https://static.wikia.nocookie.net/windows/images/8/8b/Win1983p8sm.jpg/revision/latest?cb=20220420003335)


The glow effect is a good idea, but if we have it it needs to be pixelated, e.g.
![pixel](https://art.ngfiles.com/images/1930000/1930813_anymany_neon-cactus-pixel-art.png?f1625460320)

also, take a look at the images here and how they are monochrome low color pixelated
https://solar.lowtechmagazine.com/2022/11/what-if-we-replace-guns-and-bullets-with-bows-and-arrows.html

here is another retro style website https://tilde.club/wiki/tildeverse.html



**here is some more inspiration**

![term1](https://i.pinimg.com/originals/d1/b6/55/d1b655598a89f9b9d49432dc84681faa.jpg)
![term2](https://64.media.tumblr.com/0de43fc857f6e17728d3dca9f16e514b/tumblr_oxirsm9pKI1vwnw65o1_r2_1280.gif)
![term3](https://i.etsystatic.com/30538910/r/il/aac3df/3305865168/il_1588xN.3305865168_368r.jpg)
![term4](https://preview.redd.it/kewag68yehx61.png?width=960&crop=smart&auto=webp&v=enabled&s=1ab1594cc89219d3357d5282db4bd6434b4857f1)

**Comment by Mehrabbruno** [comment]
### tasks

- update terminal_v0.0.1 - `39min`
   - move the tabs to bottom of the terminal 
   - change the color of the terminal
   - convert icon btns to be commands
   - read feedback - `7min`
   - update task and record worklog - `10min`
   - `@output` [`terminal_v0.0.2`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1211-1911&t=yzkyPlOhuzbjUi62-4)

---

### worklog

[worklog-120](https://youtu.be/uYvKP0_8uKU)

----

### feedback

---

### proposals

**Comment by serapath** [comment]
**feedback `2023.03.09`**

:sweat_smile: hmm, not sure if new or old shell was better.

## UX
1. I have trouble imagining/remembering the differences between `minimize`, `close` and `exit`
and assume having all or some of those options in a retro style tab bar like previously is actually ok, because previously the UX was great and we should strive for great UX, but wrap it in a retro appeal when it comes to ui theming. I think we should somehow stick to a retro themed more pixelated, but modern version of how windows can close/minimize/etc... to not confuse people jsut because it's retro.

2. I like the "numberered menu" items UI as a retro element, but technically we don't have more than 10 numbers to jump and also if it had many tabs you cant see the numbers anymore. You also mention the terminal title bar and i agree with what you said, maybe we can just remove it and below we change the tab names into the actual "file names" and instead of the first number, we color the first letter.

3. the focus/highlight is great, adding more tabs displaying some sort of menu to choose from in the terminal is probably cool too, but adding more tabs, closing, etc.. should probably look retro and be on the bottom, but UX wise be more like the previous terminal design, because it is just a better UX (basically also what browser tab bars do)

4. the "exit" button is probably the same as the purple terminal icon in the top menu bar that we pressed to toggle/expand the terminal, so having a little tooltip that tells people it can be activated with `esc` is probably enough, so we don't need another button in the terminal to close it and instead have a gimmick so people can actually type `"exit"` into the terminal and press `<enter>` to exit the terminal if they dont want to press the top menu bar button.



## UI

* +1 definitely the pixelated font is better.
* The almost pastel grey smooth contrast (at least to me) makes it feel more modern and pleasant, but much less retro.
* UI looks retro, but higher contrast, monochrome/low color, pixelated font, etc.. and other UI tricks for retro feel is great, but the UX should be smooth and easy, which is what i worry about a bit. the retro vibes i get from pixelated mono font, high contrasts, monochrome/low colors, maybe old school retro border/framing but potentially also from pixel dithered color filling, e.g.
  * [pixel dither](https://www.google.com/search?q=dither&tbm=isch&ved=2ahUKEwiK_Zr7pM_9AhWfsCcCHQUwCDsQ2-cCegQIABAA&oq=dither&gs_lcp=CgNpbWcQAzIECAAQQzIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDoICAAQgAQQsQNQzwNY3gdgyAhoAHAAeACAAUuIAakDkgEBN5gBAKABAaoBC2d3cy13aXotaW1nwAEB&sclient=img&ei=HgwKZIr2B5_hnsEPheCg2AM&bih=1135&biw=2194)
  * e.g. https://surma.dev/things/ditherpunk/

I think "color dither" and more retro window frame helps too in making things retro maybe.
Also, the ascii art "hello" or generally some "ascii art" text seems a good retro idea.
* also very dense and compact UI makes it feel more retro. compact UI design exists even now in modern, so compact doesn't necessarily break the UI - it is more for power users and back in retro times, that was the target audience, because it wasn't mainstream back then.
* also monochrome (not necessarily green, but often or grey or whatever but very few colors)
  * ![retro](https://www.pixelstalk.net/wp-content/uploads/images6/Green-Retro-Desktop-Wallpaper.jpg)
  * combined with "dither"
  * and pixelated
  * ![pixelated](https://cdna.artstation.com/p/assets/images/images/048/634/874/original/nela-gluhak-pc-3.gif)

Also, displays back in the days where not as crisp, so in addition to low color monochrome dither gradients, they had sometimes other artifacts and maybe subtle pixel patterns can emulate those, ...not sure if like below or similar would help anything, e.g.
* https://www.transparenttextures.com/dark-fish-skin.html
* https://www.transparenttextures.com/
* https://www.magicpattern.design/tools/css-backgrounds
* https://projects.verou.me/css3patterns/

**Comment by Mehrabbruno** [comment]
### tasks

- update terminal_v0.0.2 - `1h3min`
   - read feedback of terminal - `13min`
   - bring the v0.0.1 UX back to the updated design
   - change the color of the terminal
   - add a scrollbar
   - update task and record worklog - `13min`
   - `@output` [`terminal_v0.0.3`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1273-3879&t=If4neK4QfpiquEnl-4)

---

### worklog

[worklog-123](https://youtu.be/nVh8JEKHtbw)

----

### feedback

---

### proposals

**Comment by serapath** [comment]
**feedback `2023.03.21`**

Great.
One thing i notice though is that in terms of font size and elements, there are still too many different styles in one. retro UIs where quite homogenous in some way, because of the very limited palette of colors/fonts and stilistic elements, shapes and sizes :-)

Look at this one for example - a lot is going on there, but everything you can see somewhat feels visually like it belongs to the same style

I will prepare a few more links and pictures for inspiration of typical retro UI elements and styles we could copy/steal from :-)

![retro ui](https://camo.githubusercontent.com/677ec2024974674a34393a4fff843203bc707c6406364b7b6e4fa220ec189c88/68747470733a2f2f692e69696e666f2e637a2f696d616765732f36342f67756930342d31312e706e67)



Maybe scrollbar should stop/end before the buttons?
It confused me first, wondered what the scrollbar does. if it stopped earlier to align with tabs, before buttons/icons start, it would be more self explanatory probably.
![horizontal scrollbar](https://imgur.com/D26BJyH.png)

![scrollbar](https://imgur.com/yKC7uHV.png)
A lot of **retro scroll bars** had different arrows and styles.
e.g.: an early apple scrollbar (would prefer it)

That way we could maybe integrate the up/down arrows into the bottom menubar, on the right side of the dates dropdown?
![scrollbar](https://imgur.com/KjnontN.png)

like here (some retro scrollbars)
![scrollbars2](https://imgur.com/7GzaPHK.png)



I like the overall terminal.
One thing though is borderless "flat design" gives it a clean/modern touch.
![missing borders](https://imgur.com/i4bYm3D.png)

Most retro UI had some sort of **explicit borders.**

For example
![retro ui](https://imgur.com/0HKitkY.png)

So from previous comments here i think the following are representative:

**This one when it comes to visual elements and borders:** :star: :star: :star: 
![1](https://camo.githubusercontent.com/677ec2024974674a34393a4fff843203bc707c6406364b7b6e4fa220ec189c88/68747470733a2f2f692e69696e666f2e637a2f696d616765732f36342f67756930342d31312e706e67)

This one when it comes to monochrome.
So - while there are only 4 or 8 or 16 colors available on retro systems, in a monochrome context those can be shades of a certain color instead. So our "theme button" could maybe switch from the colored version which limits to a specific color palette of 16 colors to monochrome green or monochrome red or monochrome blue, etc... which means everything becomes 
black/white or black/green or black/red or black/blue, .... with different shades of black or green or red or blue ...
![2](https://camo.githubusercontent.com/1bc69cedc447692ae25b828babcb789b011b1dfce08d4c89108d989e52ba7d1b/68747470733a2f2f7777772e706978656c7374616c6b2e6e65742f77702d636f6e74656e742f75706c6f6164732f696d61676573362f477265656e2d526574726f2d4465736b746f702d57616c6c70617065722e6a7067)

Now if you think of the above and compare them with your current design, your design still has quite a modern touch compared to the real retro stuff :-) or not?
That's at least my impression.


Now another thing is, that almost all retro consoles/terminals/etc... where always black.
The terminal having any other color than pitch black is a rather modern thing. These days they are white and grey and transparent and whatnot, but for that reason i don't think black was a bad idea, so at least we should have under our theme button an option to go for the black terminal

![terminals](https://imgur.com/z1BdPPH.png)




The add tab button and close shell are fine, but the minimize looks like a counter part to the + so maybe people would expect to remove tabs or maybe to zoomin zoomout.
So possibly we should change the minimize/maximize icon to a more common one, like: 
![close/minimize/maximize/add buttons](https://imgur.com/0w854QT.png)

like maybe
![icons](https://imgur.com/AhfC25Z.png)




Can you also take another look at the following links and click images to take a closer look
1. https://c64os.com/c64os/usersguide/userinterface
2. http://oldmachinery.blogspot.com/2019/09/
3. http://c64.com.pl/index.php/cartridge/final.html?cartridge=finaliii
4. https://www.valoroso.it/en/the-final-cartridge-iii-tutorial-assembling-eprom-programming/
5. https://www.commodorespain.es/el-cartucho-magico-the-final-cartridge-iii/
6. https://c64os.com/post/rethinkingthememmap
7. https://c64os.com/c64os/usersguide/userinterface

They all contain specific inspirational typical retro elements and ideally we can pick the best most creative cool retro UI elements and combine them nicely :-)


### quick single picture gallery:

* check also these to see how we can make things more retro
  * http://www.symbos.de/mirrors/www.clips64.de/desktop.htm
  * http://www.symbos.de/mirrors/www.clips64.de/controlp.htm
  * http://www.symbos.de/mirrors/www.clips64.de/pathfind.htm
  * http://www.symbos.de/mirrors/www.clips64.de/processm.htm
  * http://www.symbos.de/mirrors/www.clips64.de/clipdemo.htm
* some real retro screenshots (**you might need to zoom in on some**)
  * https://commodore.software/images/jdownloads/screenshots/cbm%20command%20v2.3%201.jpg
  * https://commodore.software/images/jdownloads/screenshots/dracopy-1.3doj-128-1.jpg
  * https://commodore.software/images/jdownloads/screenshots/dracopy%20v1.0d-1.png
  * https://commodore.software/images/jdownloads/screenshots/ez-loader-v2.6-1.jpg
  * https://commodore.software/images/jdownloads/screenshots/fibr-1.jpg
  * https://commodore.software/images/jdownloads/screenshots/nav%20v.9-1.jpg
  * https://commodore.software/images/jdownloads/screenshots/worlds%20greatest%20menu.jpg
  * https://www.nightfallcrew.com/wp-content/uploads/2015/12/sidplay100.jpg?2b7c37
  * https://imgur.com/lr2r5XW.png
  * https://www.myabandonware.com/media/screenshots/m/mission-omega-5zx/mission-omega_2.png
  * https://www.myabandonware.com/media/screenshots/m/mission-omega-5zx/mission-omega_3.png
  * https://www.myabandonware.com/media/screenshots/m/mission-omega-5zx/mission-omega_4.png

**Comment by Mehrabbruno** [comment]
### tasks - `2023.03.29`

- update terminal_v0.0.3 - `28min`
   - update the buttons and the icons
   - add border and change the color of the terminal tabs
   - update the scrollbar
   - read feedback - `23min`
   - update task and record worklog - `14min`
   - `@output` 📦 [`terminal_v0.0.4`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1315-1911&t=oHWrNNsurk7EuHVg-4)

---

### worklog

[worklog-125](https://youtu.be/jYZ8dfopn9w)

----

### feedback

---

### proposals

**Comment by serapath** [comment]
**feedback `2023.03.29`**

You make a good point with containers in the content area
![screen](https://imgur.com/Z0aPKCI.png)

I think we definitely need a **focus color** (maybe orange or pink or green)
* to use in background colors, borders or font and icon colors **only when focus**

ALSO:

Below i will mix some retro screenshots with the screenshots from our current design.
Seeing them side by side definitely shows that our system still looks way more modern and things aren't right yet. Maybe you can medidate with those and study and get just even more in into the retro vibe to really feel it? :-) Not saying what we have is bad, but I just think it can get even better.

I kinda wish we could homogenize the style and really keep it even more retro.
everything has a more standardized layout along a mono font character sized imaginary grid.

I think things get better slowly, but don't feel shy to put some serious hours in and also standardize everything more into one retro style. I will attach again some more pictures.
Currently fonts, menubars, icons and colors are still not yet in this grid like retro style.


Just to keep in mind, users will anyway use their scroll wheel or maybe click the scrollbar and drag it, nobody really uses the scroll bar arrows anyway, they are more a "visual gimmick" to make it appear more retro.

For example scrollbar
![scrollbar](https://imgur.com/vtuNpWV.png)



Basically those are also very representative. mono color and again everything is compact and aligned.


Here is that applied to the terminal, but of course, i am not good in making it look more retro, like the image above, but still, just to share what i mean:
![screen](https://i.imgur.com/GZXbgZZ.png)
![screen2](https://i.imgur.com/Ar0MiRA.png)

the following also has a lot more retro touch, if that is possible to achieve.
Whether it is the scrollbar from this one or the one above from the beginning.
This one has the dotted pattern and the other one has the diagonal striped pattern.
The point is that the entire UI of all screens uses the same retro style.

![retro ui](https://camo.githubusercontent.com/677ec2024974674a34393a4fff843203bc707c6406364b7b6e4fa220ec189c88/68747470733a2f2f692e69696e666f2e637a2f696d616765732f36342f67756930342d31312e706e67)

This one for example is even older and even more retro, so if you can even do that, that would be cool

![retro ui](https://imgur.com/0HKitkY.png)


### I just think the whole thing needs some **retro :heart: :heart: :heart:** :slightly_smiling_face: 

some more real retro screenshots (you might need to zoom in on some)
![a1](https://commodore.software/images/jdownloads/screenshots/cbm%20command%20v2.3%201.jpg)
![a2](https://commodore.software/images/jdownloads/screenshots/dracopy-1.3doj-128-1.jpg)
![a3](https://commodore.software/images/jdownloads/screenshots/dracopy%20v1.0d-1.png)
![a4](https://commodore.software/images/jdownloads/screenshots/ez-loader-v2.6-1.jpg)
![a5](https://commodore.software/images/jdownloads/screenshots/fibr-1.jpg)
![a6](https://commodore.software/images/jdownloads/screenshots/nav%20v.9-1.jpg)
![a7](https://commodore.software/images/jdownloads/screenshots/worlds%20greatest%20menu.jpg)
![a8](https://www.nightfallcrew.com/wp-content/uploads/2015/12/sidplay100.jpg?2b7c37)
![a9](https://imgur.com/lr2r5XW.png)
![a10](https://www.myabandonware.com/media/screenshots/m/mission-omega-5zx/mission-omega_2.png)
![a11](https://www.myabandonware.com/media/screenshots/m/mission-omega-5zx/mission-omega_3.png)
![a12](https://www.myabandonware.com/media/screenshots/m/mission-omega-5zx/mission-omega_4.png)



Basically those links help:

Can you also take another look at the following links and click images to take a closer look
https://c64os.com/c64os/usersguide/userinterface
http://oldmachinery.blogspot.com/2019/09/
http://c64.com.pl/index.php/cartridge/final.html?cartridge=finaliii
https://www.valoroso.it/en/the-final-cartridge-iii-tutorial-assembling-eprom-programming/
https://www.commodorespain.es/el-cartucho-magico-the-final-cartridge-iii/
https://c64os.com/post/rethinkingthememmap
https://c64os.com/c64os/usersguide/userinterface
They all contain specific inspirational typical retro elements and ideally we can pick the best most creative cool retro UI elements and combine them nicely :-)


check also these to see how we can make things more retro
http://www.symbos.de/mirrors/www.clips64.de/desktop.htm
http://www.symbos.de/mirrors/www.clips64.de/controlp.htm
http://www.symbos.de/mirrors/www.clips64.de/pathfind.htm
http://www.symbos.de/mirrors/www.clips64.de/processm.htm
http://www.symbos.de/mirrors/www.clips64.de/clipdemo.htm

**Comment by Mehrabbruno** [comment]
### tasks - `2023.04.08`

- update terminal_v0.0.4 - `2h30min`
   - read feedback on terminal - `12min`
   - update the UI to be flat retro
   - `@output` 📦 [`terminal_v0.0.5`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1359-2762&t=Ox51Ohm1m5cMB0nq-4)

- update terminal_v0.0.5 - `24min`
   - add the scrollbar_v0.0.2. - `5min`
   - change the highlight colors
   - make the buttons more compacts
   - record and update tasks, worklog etc - `20min`
   -  `@output` 📦 [`terminal_v0.0.6`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1359-3226&t=Ox51Ohm1m5cMB0nq-4)

---

### worklog

[worklog-129](https://youtu.be/r3M3Tr8WWf0)

----

### feedback

---

### proposals

**Comment by serapath** [comment]
**feedback `2023.04.09`**

Great. At first i was sad for the terminal to not have the pink window bar anymore.
I wish we could have one theme version where we actually keep this :P
...But It makes sense, because we can use colors for highlighting messages in the terminal.

Also the more squeezed font in the terminal tabs makes it look more retro. that's cool.

I also like the renaming of `terminal` into `shell` after seeing you do it in the video.
Of course - the black window bar should be the main theme color from light or dark theme, where the default one has the dat dark_blue

So all great.
Regarding icons, maybe one thing:
   * let's remove the `min/max` icon for now to keep it simple
   * maybe we can or should move the `+` button into the tabs row
   * and thus make the scrollbar under the `+`.
   * the `<` and `>` button of the scrollbar might have the same width as the `+` maybe.
![terminal](https://imgur.com/rj6TXv9.png)


Basically, something that makes sens to align all windows, scrollbars, icons etc... in an invisible grid to be multiplze sizes of a basica square and aligned to each other, so that is the main goal - currently we still have `+/#/x` buttons not squares but somewhat rectangles of an in between height not fitting into an imaginary grid.

if we later "move" windows around or icons, they should probably snap from one grid unit to the next instead of being continuously dragged - but that needs to wait until we actually implement it, but it also means nothing can be resized or positioned in between those invisible square grid blocks

**Comment by Mehrabbruno** [comment]
### tasks - `2023.04.14`

- update terminal_v0.0.6 - `31min`
   - read feedback on terminal - `4min`
   - remove the extra buttons from the bottom tab container
   - record and update tasks, worklog etc - `11min`
   - `@output` 📦 [`terminal_v0.0.7`](https://www.figma.com/file/1zBxgchAmrDGw2FDzu7uxX/dat-ecosystem-website?node-id=1430-2686&t=idhOGERipmYR9EQg-4)

---

### worklog

[worklog-132](https://youtu.be/Y53KRwfx2dw)

----

### feedback

---

### proposals

**Comment by serapath** [comment]
**feedback `2023.04.17`**

+1 perfect changes. i think the terminal is great.
